tests/Http/PostTest.php

```diff
@@ -1628,4 +1628,51 @@ public function testCreatingVoterRegistrationPost()
             'details' => json_encode($details),
         ]);
     }
+
+    /**
+     * Test that when a post is created for a signup with a group_id
+     * the group_id is saved to the post as well.
+     *
+     * @return void
+     */
+    public function testCreatingAPostForSignupWithGroupId()
+    {
+        $groupId = factory(Group::class)->create()->id;
+        $signup = factory(Signup::class)->create(['group_id' => $groupId]);
+
+        // Attributes for the post that we'll create
+        $quantity = $this->faker->numberBetween(10, 1000);
+        $text = $this->faker->sentence;
+        $action = factory(Action::class)->create(['campaign_id' => $signup->campaign_id]);
+
+        // Mock the Blink API call.
+        $this->mock(Blink::class)->shouldReceive('userSignupPost');
+
+        // Create the post!
+        $response = $this->withAccessToken($signup->northstar_id)->postJson('api/v3/posts', [
+            'northstar_id'     => $signup->northstar_id,
+            'campaign_id'      => $signup->campaign_id,
+            'type'             => $action->post_type,
+            'action'           => $action->name,
```

Comment:
I don't think we need this property... I believe there's an ancient TODO to deprecate support for this `action` parameter once all Contentful entries have an `action_id` field (and can't find the related cleanup Pivotal story)

Reply:
Oh great eye @aaronschachter ! I missed that there was `action` and `action_id` here